starlight.spec

```diff
@@ -0,0 +1,35 @@
+Release:	1%{?dist}
```

Comment:
This definition is unused, it should instead read:
```### RPM external starlight r193``` (using the current release on cms-externals repository)